examples/example_lib_input_string.ks

```diff
@@ -0,0 +1,53 @@
+// This file is distributed under the terms of the MIT license, (c) the KSLib team
+
+// This is intended to be run as a boot script.
+
+run lib_input_string.
+run lib_file_exists.
+run spec_char.
+
+if not file_exists("acount.ksm") {
+ set newuser to true.
+} else {
+ set newuser to false.
+ run acount.ksm.
```

Comment:
*account

Not sure if this relies on a specific script; if so, this is a minor spelling error that can be ignored